.changeset/warm-fans-smile.md

```diff
@@ -0,0 +1,5 @@
+---
+'@ithaka/pharos': minor
```

Comment:
nit - Is this a bugfix more than minor improvement?

Reply:
I wouldn't think so? This is not a breaking change so it's not major. Is there something more appropriate? Patch?

Reply:
Bug fixes should be patches, yes.